### PR TITLE
GSW-1381 fix minor issues related to earn

### PR DESCRIPTION
**Pull request by AkydA**
Description
---
- earn page: remove redundant token icon in incentives
- wallet page: token link text
- tooltip: fix token symbol text
- incentivize : update disclaimer
- incentivize : missing snackbar text
- translation: swap notification
- translation: error page

### CODE REVIEW

File: packages/web/public/locales/de/IncentivizePool.json [hunk 1]
```diff
@@ -1 +1 @@
...
...e":"Haftungsausschluss"},"incenDetail":{"row":{"label":{"period":"Zeitraum","pool":"Pool","totalAmt":"Gesamtbetrag"},"value":{"period":{"desc":"{{symbol}} werden täglich verteilt"}}}},"incentiPool":{"form":{"header":"Anreize für Pools","period":{"field":{"disPeriod":{"label":"Verteilungszeitraum","tootlip":"Die Anreize werden auf der Grundlage der Anzahl der Blöcke verteilt. Der Verteilungszeitraum ist ein Näherungswert und kann aufgrund der Netzwerkbedingungen der Blockchain leicht von der tatsächlichen Zeit abweichen."},"startDate":{"label":"Datum des Beginns"}},"label":"2. Verteilungszeitraum auswählen"},"pool":{"col":{"liqui":"Liquidität","pools":"Schwimmbäder"},"label":"1. Pool auswählen","searchPlaceHolder":"Suche nach Name, Symbol oder Pfad"},"rewaAmt":{"label":"3. Belohnungsbetrag festlegen"}}},"liquidity":"Liquidität","rewards":"Belohnungen","selectToken":"Ein Token auswählen","setReward":"Belohnungsbetrag festlegen","submitBtn":{"amtTooLow":"Zu niedriger Betrag","enterAmt":"Betrag eingeben","incentiPool":"Anreize für Pools","insuffi":"Unzureichendes Gleichgewicht","selectPool":"Pool auswählen","switch":"Nach Gnoland wechseln","walletLoginBtn":"Anmeldung zur Brieftasche"}}
```
Comment (notJoon) [inline packages/web/public/locales/de/IncentivizePool.json:1]: I think `tootlip` is a typo for `tooltip`, please Check

File: packages/web/public/locales/fr/IncentivizePool.json [hunk 2]
```diff
@@ -1 +1 @@
...
...":{"period":"Période","pool":"Piscine","totalAmt":"Montant total"},"value":{"period":{"desc":"{{symbol}} sera distribué quotidiennement"}}}},"incentiPool":{"form":{"header":"Incitation à la mise en commun","period":{"field":{"disPeriod":{"label":"Période de distribution","tootlip":"Les incitations sont distribuées en fonction du nombre de blocs. La période de distribution est approximative et peut légèrement différer de la durée réelle en fonction des conditions du réseau de la blockchain."},"startDate":{"label":"Date de début"}},"label":"2. Sélectionner la période de distribution"},"pool":{"col":{"liqui":"Liquidité","pools":"Piscines"},"label":"1. Sélectionnez Pool","searchPlaceHolder":"Recherche par nom, symbole ou chemin"},"rewaAmt":{"label":"3. Fixer le montant de la récompense"}}},"liquidity":"Liquidité","rewards":"Récompenses","selectToken":"Sélectionner un jeton","setReward":"Fixer le montant de la récompense","submitBtn":{"amtTooLow":"Montant trop faible","enterAmt":"Saisir le montant","incentiPool":"Incitation à la mise en commun","insuffi":"Solde insuffisant","selectPool":"Sélectionner une piscine","switch":"Passer à Gnoland","walletLoginBtn":"Connexion au portefeuille"}}
```
[the remark this answers is not in the document]
Comment (AkydA) [reply packages/web/public/locales/fr/IncentivizePool.json:1]: Report to Jaeho please.

[the remark this answers is not in the document]
Comment (notJoon) [reply packages/web/public/locales/fr/IncentivizePool.json:1]: Additionally, the desc part is missing a definite article. 
`"desc":"{{symbol}} sera distribué quotidiennement"` to "desc": "Le {{symbol}} sera distribué quotidiennement".

[the remark this answers is not in the document]
Comment (notJoon) [reply packages/web/public/locales/fr/IncentivizePool.json:1]: Additionally, the `desc` field is missing a definite article. 
`"desc":"{{symbol}} sera distribué quotidiennement"` to `"desc": "Le {{symbol}} sera distribué quotidiennement"`.

File: packages/web/public/locales/ja/IncentivizePool.json [hunk 3]
```diff
@@ -1 +1 @@
...
...インセンティブ・プールを確認する","row":{"label":{"period":"期間","pool":"プール","totalAmt":"総額"},"title":"報酬","value":{"period":{"desc":"{{amount}} {{symbol}} 毎日配布される"}}},"title":"インセンティブ・プールを確認する"},"disclaimer":{"description":"この機能により、特定の流動性プールに対するステーキング報酬としてインセンティブを提供することができます。先に進む前に、外部インセンティブの仕組みとウォームアップ期間<docs_link/> を理解していることを確認してください。ウォームアップ期間からの未分配額は、インセンティブ付与期間の完了時にインセンティブ提供者のアドレスに送信されます。<br /><br />このステップを完了すると、報奨を撤回することはできません。追加した報奨は、契約により自動的に分配されます。","title":"免責事項"},"incenDetail":{"row":{"label":{"period":"期間","pool":"プール","totalAmt":"総額"},"value":{"period":{"desc":"{{symbol}} 毎日配布される"}}}},"incentiPool":{"form":{"header":"プールを奨励する","period":{"field":{"disPeriod":{"label":"配信期間","tootlip":"インセンティブはブロック数に応じて分配されます。配布期間はおおよその目安であり、ブロックチェーンのネットワーク状況により実際の配布期間と若干異なる場合があります。"},"startDate":{"label":"開始日"}},"label":"2.配信期間を選択"},"pool":{"col":{"liqui":"流動性","pools":"プール"},"label":"1.プールを選択","searchPlaceHolder":"名前、記号、パスで検索"},"rewaAmt":{"label":"3.報酬額の設定"}}},"liquidity":"流動性","rewards":"報酬","selectToken":"トークンを選択","setReward":"報酬額の設定","submitBtn":{"amtTooLow":"金額が低すぎる","enterAmt":"金額を入力","incentiPool":"プールを奨励する","insuffi":"残高不足","selectPool":"プール選択","switch":"グノランドへ移動","walletLoginBtn":"ウォレットログイン"}}
```
Comment (notJoon) [inline packages/web/public/locales/ja/IncentivizePool.json:1]: `"プールを奨励する"` -> `"プールにインセンティブを付与する"`

The former sentence means "Encourage pools" (장려하다)and the latter means "Provide incentives to pools", so I think this is a better translation. It also more accurately shows the specific reward mechanism for pools.

**Review by jinoosss — APPROVED** [review]

**Review by notJoon — APPROVED** [review]

**Pull request merged**